screen/store/components/pages/CheckOutPage/CheckOutPage.js

```diff
@@ -157,6 +248,17 @@ var CheckOutPage = {
         this.getCartInfo();
       }.bind(this));
     },
+    selectBillingAddress(address) {
```

Comment:
Cant this be done with object assign?
```
this.paymentMethod = Object.assign(this.paymentMethod, address);
```

Why is a variable called "paymentMethod" containing an address?